packages/common/src/getContract.ts

```diff
@@ -98,11 +108,11 @@ export function getContract<
               args,
               ...options,
               onWrite,
-            } as unknown as WriteContractParameters<TAbi, typeof functionName, TChain, TAccount>;
+            } as unknown as WriteContractParameters;
```

Comment:
are we losing type specificity here?

Reply:
This loses some type specificity. I'll fix it.